src/views/pam/Integration/components/AccountList.vue

```diff
@@ -22,8 +22,10 @@ export default {
     return {
       tableConfig: {
         url: `/api/v1/accounts/accounts/?integrationapplication=${this.object.id}`,
-        hasLeftActions: true,
-        hasImport: true,
+        hasLeftActions: false,
+        hasImport: false,
+        hasExport: false,
+        showQuickFilters: false,
         columnsMeta: {
           asset: {
             formatter: DetailFormatter,
```

Comment:
There are no apparent differences beyond the difference of `hasLeftActions` and `hasImport`. Both indicate that there's at least one action on left side and that export is not enabled.

So overall, here is my suggestion:

The code works well without changes!